lib/scala/akka-native/src/main/java/org/enso/nativeimage/workarounds/ReplacementAkkaSpinWait.java

```diff
@@ -0,0 +1,18 @@
+package org.enso.nativeimage.workarounds;
```

Comment:
Is there a reason we've got our own copy of akka-native?

Reply:
Do you mean this library https://github.com/vmencik/akka-graal-native ?

If I recall correctly it was made for a different version of Akka so there was a class that it set-up which did not exist causing errors (or the other way around or both ways). Given the amount of changes and that we needed to trace the reflective accesses anyway, I though it will be simpler to just have our own implementation of these workarounds.

Reply:
Ah okay! Works for me.